moto_week3/s3_func_test.py

```diff
@@ -0,0 +1,72 @@
+import boto3
+import io
+import sys
+import unittest
+from moto import mock_s3
+from s3_functions import main, get_client, list_s3_buckets, list_s3_objects, read_s3_object
+
+
+class S3Tests(unittest.TestCase):
+    def setUp(self):
+        #sets up the local moto s3 service for mocking.
+        self.bucket = 'buck02jay'
+        self.key = 'upload.txt'
+        self.value = 'RA1511001010001'
+
+    @mock_s3
+    def __moto_setup(self):

```

Comment:
private method begins with single underscore character (PEP8)
```suggestion
    def _moto_setup(self):
```